generators/beamAndVertexGenerator.h

```diff
@@ -61,22 +80,48 @@ namespace rpwa {
 		bool _readBeamfileSequentially;
 		long _currentBeamfileEntry;
 
+		virtual bool loadBeamFileWithMomenta();
```

Comment:
would be nice to add brief comments describing what each function does/is needed for